packages/x-components/src/x-modules/empathize/components/__tests__/empathize.spec.ts

```diff
@@ -1,53 +1,110 @@
-import { mount } from '@vue/test-utils';
+import { mount, Wrapper } from '@vue/test-utils';
 import Vue from 'vue';
-import BaseEventButton from '../../../../components/base-event-button.vue';
 import { getXComponentXModuleName, isXComponent } from '../../../../components/x-component.utils';
 import { getDataTestSelector, installNewXPlugin } from '../../../../__tests__/utils';
 import Empathize from '../empathize.vue';
+import { empathizeXModule } from '../../x-module';
+import { XPlugin } from '../../../../plugins/index';
+import { XEvent } from '../../../../wiring/index';
+
+jest.useFakeTimers();
 
 describe('testing empathize component', () => {
-  let localVue: typeof Vue;
+  function renderEmpathize({
+    eventsToOpenEmpathize = ['UserClickedSearchBox'],
+    eventsToCloseEmpathize = ['UserClosedEmpathize'],
+    template = `<Empathize v-bind="$attrs">
+                  <template #default>
+                    <span data-test="empathize-content">Empathize</span>
+                  </template>
+                </Empathize>`
+  }: RenderEmpathizeOptions = {}): RenderEmpathizeAPI {
+    const [, localVue] = installNewXPlugin();
+    XPlugin.registerXModule(empathizeXModule);
 
-  beforeEach(() => {
-    [, localVue] = installNewXPlugin({});
-  });
+    const parent = document.createElement('div');
+    document.body.appendChild(parent);
+
+    const wrapper = mount(
+      {
+        components: { Empathize },
+        template
+      },
+      {
+        localVue,
+        attachTo: parent,
+        propsData: {
+          eventsToOpenEmpathize,
+          eventsToCloseEmpathize
+        }
+      }
+    ).findComponent(Empathize);
+
+    return {
+      wrapper
+    };
+  }
 
   it('is an XComponent which has an XModule', () => {
-    const empathize = mount(Empathize, { localVue });
-    expect(isXComponent(empathize.vm)).toEqual(true);
-    expect(getXComponentXModuleName(empathize.vm)).toEqual('empathize');
+    const { wrapper } = renderEmpathize();
+    expect(isXComponent(wrapper.vm)).toEqual(true);
+    expect(getXComponentXModuleName(wrapper.vm)).toEqual('empathize');
+  });
+
+  it('will not open empathize if there is no content to render', async () => {
```

Comment:
You can safely remove the `async` modifier as it is not needed for this test

```suggestion
  it('will not open empathize if there is no content to render', () => {
```